OracleSiebel/OEL/build/multi-stage/sai/dockerfile

```diff
@@ -0,0 +1,108 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+#
+# Since: Dec, 2019
+# Authors: andreas.thieltges@oracle.com and duncan.ford@oracle.com
+# Description: Build Siebel SAI container
+#
+# Notes: This build is execute from the build-all-multistage script
+#
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+
+# Globals - these are overridden by the calling script
+
+ARG VERSION=19.11
+ARG BUILD_DATE=2019-12-06
+
+
+FROM oraclelinux:latest as temp
+#FROM oraclelinux:7-slim as temp
+ARG INSTALLSITE=installers
+ARG VERSION
+
+COPY oraInst.loc getBase.sh getUpdate.sh sai/sai_base.rsp sai/sai_install_base.sh sai/sai_update.rsp.templ sai/sai_install_update.sh siebelkeystore.jks /config/
+
+# Install Siebel Base
+
+ENV LANG=en_US.UTF-8
+
+RUN mkdir -p -ma+rwx /siebel/sai /siebel/oraInventory /stage \
+&& groupadd -g 1000 siebel && useradd -m -g siebel -G siebel -u 1000 siebel \
+&& yum -y install wget tcsh ksh \
+&& cd /mnt \
+&& date \
+&& echo Downloading installer \
+&& bash /config/getBase.sh ${INSTALLSITE} \
+&& chmod a+rx /config \
+&& chown -R siebel:siebel /config \
+&& date \
+&& chmod +x /mnt/Siebel_Enterprise_Server/Disk1/install/unzip \
+&& chmod +x /mnt/Siebel_Enterprise_Server/Disk1/install/runInstaller \
+&& cd .. \
+&& echo Installing Siebel ... \
+&& bash /config/sai_install_base.sh \
+&& echo Waiting for deployment of web archive ... \
+&& sleep 30 \
+&& mkdir /siebel/sai/ps_backup/ \
+&& chown -R siebel:siebel /siebel \
+&& chmod -R 755 /siebel/sai
+
+# Install Siebel Update
+
+RUN cd /mnt \
+&& date \
+&& echo Downloading installer \
+&& bash /config/getUpdate.sh ${INSTALLSITE} ${VERSION} \
+&& chmod a+rx /config \
+&& chown -R siebel:siebel /config \
+&& date \
+&& chmod +x /mnt/Siebel_Enterprise_Server/Disk1/install/unzip \
+&& chmod +x /mnt/Siebel_Enterprise_Server/Disk1/install/runInstaller \
+&& cd .. \
+&& bash /config/sai_install_update.sh \
+&& echo Waiting for deployment of web archive ... \
+&& sleep 30 \
+&& rm -rf /mnt/Siebel_Enterprise_Server/ \
+&& rm -rf /siebel/sai/ps_backup/ \
+&& chmod -R 755 /siebel/sai
+
+# Create Final Container
+
+FROM oraclelinux:latest
```

Comment:
Please switch to `oraclelinux:7-slim` (preferred) or at least `oraclelinux:7` as `oraclelinux:latest` will switch to Oracle Linux 8 fairly soon

Reply:
Done